main.py

```diff
@@ -0,0 +1,45 @@
+# This is Italian Card Game Scopa
+# Scopa rules can be found here https://en.wikipedia.org/wiki/Scopa
+
+import sys
+import uipyqt
+import scopa
+from PyQt5.QtWidgets import *
```

Comment:
Sonarlink: Import only needed names or import the module and then use its members.